### PR TITLE
Update README-pt.md

### PR DESCRIPTION
**Pull request by jailton**
Update README-pt.md following the latest README.md English version.

### CODE REVIEW

File: README-pt.md [hunk 1]
```diff
@@ -1,76 +1,137 @@
...
+git clone https://github.com/kubernetes/website.git
+cd website
+```
 
-A maneira recomendada de executar o site do Kubernetes localmente é executar uma imagem especializada do [Docker](https://docker.com) que inclui o gerador de site estático [Hugo](https://gohugo.io).
+O website do Kubernetes utiliza o [tema Docsy Hugo](https://github.com/google/docsy#readme). Mesmo se você planeje executar o website em um container, é altamente recomendado baixar os submódulos e outras dependências executando o seguinte comando:
 
-> Se você está rodando no Windows, você precisará de mais algumas ferramentas que você pode instalar com o [Chocolatey](https://chocolatey.org). `choco install make`
+```
+# pull in the Docsy submodule
```
Comment (sftim) [inline README-pt.md:30]: If you want, you can also translate this comment.

**Comment by jailton** [comment]
@sftim done.

File: README-pt.md (hunk 1 above)
Comment (jailton) [reply README-pt.md]: Comment translated.

**Comment by sftim** [comment]
echoing https://github.com/kubernetes/website/pull/26006#issuecomment-768945817
/approve

**Pull request merged**